homeassistant/components/knx/weather.py

```diff
@@ -64,3 +64,10 @@ def wind_speed(self):
             if self._device.wind_speed is not None
             else None
         )
+
+    @property
+    def wind_bearing(self):
+        """Return current wind bearing in degrees."""
+        return (
+            self._device.wind_bearing if self._device.wind_bearing is not None else None
+        )
```

Comment:
```suggestion
        return self._device.wind_bearing
```

Maybe I'm missing something here, but isn't this functionally the same?

Case: `None if None is not None else None`

Reply:
Yes, you are right. Same for humidity. I have corrected this. Thanks!